lte/gateway/deploy/roles/agw_docker/tasks/main.yml

```diff
@@ -63,7 +63,7 @@
   template:
     src: env.j2
     dest: /var/opt/magma/docker/.env
-    mode: 0400
+    mode: 0644
```

Comment:
That makes the `DOCKER_PASSWORD={{ docker_pass }}` world readable and makes provisioned files editable

Reply:
Updated the PR